### PR TITLE
Feature/create gym plan page

### PR DESCRIPTION
**Pull request by KirstenSchuhmann**
Created PR to share code & error with Antje. (Danke Antje fürs aushelfen!!  ☺️)

### CODE REVIEW

**Comment by KirstenSchuhmann** [comment]
PR refers to this [User Story](https://github.com/users/KirstenSchuhmann/projects/3?pane=issue&itemId=17606968). 
 
### Updated files & info about code: 
- Added Link to each gymPlan title in **index.js**
-  dynamic Routing to [id].js -> SavedWorkoutRoutine.js  (Mapping through the a gymPlans) 
- Added Styling to **SavedWorkoutRoutine.js**
-  _AllSavedWorkoutRoutine.js is currently not used anywhere_, can be ignored for code review, because it will change anyway in a different US. 
- Folder -> CreateGymPlan has now new file **SectionAndExerciseSetStyling.js** Styling the Section & ExerciseSet of the Form" 
- Added Styling to the **index.file**

File: pages/[id].js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import SavedWorkoutRoutine from '../components/SavedWorkoutRoutine/SavedWorkoutRoutine';
+
+export default function OverviewOfAddedRoutines({ gymPlans = [] }) {
+   const router = useRouter();
+   const { id } = router.query;
+
+   // Creates new Array of the addressed gymPlan
+
+   if (!gymPlans) {
+      return <p> Geht grad nicht 🤓 </p>;
```
Comment (AntjeWalter) [inline pages/[id].js:15]: Maybe in English because your app is in English ;D

File: pages/_app.js [hunk 2]
```diff
@@ -1,12 +1,55 @@
...
+         id: nanoid(),
+         sets: '',
+         reps: '',
+         weight: '',
+         exercise: '',
+      });
+      setSections(() => updatedSections);
+   }
+
+   // Delete Section
```
Comment (AntjeWalter) [inline pages/_app.js:30]: I think your function is named well enough so you do not need this comment :)

File: components/CreateGymPlan/CreateWorkoutRoutineForm.js [hunk 3]
```diff
@@ -103,20 +93,24 @@ export default function GymPlan({ onCreatePlan }) {
...
             sections={sections}
-            addSection={addSection}
-            addExerciseSet={addExerciseSet}
-            deleteSection={deleteSection}
+            onAddExerciseSet={onAddExerciseSet}
+            onDeleteSection={onDeleteSection}
          />
+         <SaveButton
+            type='submit'
+            aria-label='add workout routine'>
```
[the remark this answers is not in the document]
Comment (KirstenSchuhmann) [reply components/CreateGymPlan/CreateWorkoutRoutineForm.js:111]: You're right, I think I'm gonna remove it, but I'll check it with the screen reader, just to see what happens when the information is provided twice.

**Pull request merged**